doc/install_ubuntu.txt

```diff
@@ -1,30 +1,33 @@
-:orphan:
-
 .. _install_ubuntu:
 
-Easy Installation of an Optimized Theano on Current Ubuntu
-==========================================================
 
-For NVIDIA Jetson TX1 embedded platform:
+Ubuntu Installation Instructions
+================================
 
-.. code-block:: bash
+There are two ways of getting the dependencies for Theano. You can either grab them
+as system packages or let pip do everything for you. 
 
-    sudo apt-get install python-numpy python-scipy python-dev python-pip python-nose g++ libblas-dev git
-    pip install --upgrade --no-deps git+git://github.com/Theano/Theano.git --user  # Need Theano 0.8(not yet released) or more recent
+There are also two installation modes, the stable version and the bleeding-edge most recent version.
+
+Requirements through System Packages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to acquire the requirements through your system packages follow these instructions:
 
 For Ubuntu 11.10 through 14.04:
 
 .. code-block:: bash
 
     sudo apt-get install python-numpy python-scipy python-dev python-pip python-nose g++ libopenblas-dev git
-    sudo pip install Theano
 
 For Ubuntu 11.04:
 
 .. code-block:: bash
 
     sudo apt-get install python-numpy python-scipy python-dev python-pip python-nose g++ git libatlas3gf-base libatlas-dev
-    sudo pip install Theano
+
+.. include:: install_generic.txt
```

Comment:
The text doesn't get coloured because you don't include the `<style>` instructions.
You can fix it by adding (before this line):
```
.. include:: install_generic.txt
    :end-line: 9
```

I suggest you put the style instructions in a separate file (call it css.txt) and just include it at the beginning of each txt file where you have coloured instructions.